saq/worker.py

```diff
@@ -341,6 +340,13 @@ def start(
     loop = asyncio.new_event_loop()
     worker = Worker(**settings_obj)
 
+    async def worker_start() -> None:
```

Comment:
why did you move this? isn't worker.stop already called on the finally of def start?

Reply:
I wanted to decouple queue connect/disconnect from worker start/stop. That way, if there are multiple workers sharing the same queue, stopping one won't disconnect the queue for the others. This happens in the tests, for example.